pkgs/applications/editors/kakoune.cr/default.nix

```
@@ -0,0 +1,42 @@
+{ bat, crystal, fd, fzf, fetchFromGitHub, fetchurl, jq, lib, stdenv }:
+let
+  icon = fetchurl {
+    url = "https://github.com/mawww/kakoune/raw/master/doc/kakoune_logo.svg";
+    sha256 = "1x9gdrnrqgzvr1ixr6s8ff6cvz01yp2xdh5ad6nbh6p2d094h617";
+    name = "kakoune.svg";
+  };
+
+in crystal.buildCrystalPackage {
+  pname = "kakoune-cr";
+  version = "0.1.0";
+
+  dependencies = [ bat fd fzf jq ];
+
+  src = fetchFromGitHub {
+    repo = "kakoune.cr";
+    owner = "alexherbo2";
+    rev = "48d2875c743c9180e40e372ed06a3abd21a4683a";
+    sha256 = "0l34cw6vzd87ivm7b7n8ry3mivi50pgjkklpapv766zgdpr2snip";
```

```suggestion
    rev = "c0bfd9e76a972359098af9f1957ab681e3a54318";
    sha256 = "sha256-acqC5RT9duI/zmE5Nk7COI2sQzvi0lkmLJ8orvBOL7c=";
```
Updated while I was at it.